src/Data/ProgrammesDb/Entity/Clip.php

```diff
@@ -2,11 +2,13 @@
 
 namespace BBC\ProgrammesPagesService\Data\ProgrammesDb\Entity;
 
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\Entity\EntityInterfaces\GalleriesCountableInterface;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
  * @ORM\Entity()
  */
-class Clip extends ProgrammeItem
+class Clip extends ProgrammeItem implements GalleriesCountableInterface
 {
+    use Traits\AggregatedGalleriesCountMethodsTrait;
```

Comment:
Can this be imported?

Reply:
We are using traits like that in few places, I think importing them is a more elegant, thanks.